app/src/test/java/com/kickstarter/viewmodels/BackingAddOnViewHolderViewModelTest.kt

```diff
@@ -158,11 +162,34 @@ class BackingAddOnViewHolderViewModelTest : KSRobolectricTestCase() {
         this.disableIncreaseButton.assertValues(false, false, false, true)
     }
 
+    @Test
+    fun increaseStepperDisable_WhenAddOnUnavailable() {
```

Comment:
nit: I would rename this test method to: `disableIncreaseButton_whenAddOnUnavailble_returnsTrue()`